watertap/unit_models/ion_exchange_0D.py

```diff
@@ -0,0 +1,1352 @@
+            doc="Concentration of regen solution [kg regen/m3 soln]",
+        )
+
+        self.regen_bv = Var(
+            initialize=2,
+            units=pyunits.dimensionless,
+            doc="Regeneration volume required per BV",
+        )
+
+        self.regen_flow = Var(
+            initialize=2,
+            units=pyunits.m**3 / pyunits.s,
+            doc="Regeneration flow rate",
+        )
+
+        self.t_regen = Var(initialize=30, units=pyunits.s, doc="Regeneration time")
+
+        # # ====== Backwashing ====== #
+
+        self.bw_rate = Var(
+            initialize=5,
+            units=pyunits.m / pyunits.hour,
+            bounds=(4.5, 6.5),
+            doc="Backwash rate [m/hr]",
+        )
+
+        self.bw_flow = Var(
+            initialize=0.001,
+            units=pyunits.m**3 / pyunits.s,
+            doc="Backwashing flow rate",
+        )
+
+        self.t_bw = Var(
+            initialize=360, units=pyunits.s, bounds=(120, 1200), doc="Backwash time"
+        )
+
+        self.bed_expansion_frac = Var(
+            initialize=0.5,
+            bounds=(0.4, 1.2),
+            units=pyunits.dimensionless,
+            doc="Fraction of bed depth increase during backwashing",
+        )
+
+        self.bed_expansion_h = Var(
+            initialize=0.5, units=pyunits.m, doc="Bed depth increase during backwashing"
+        )
+
+        # # ====== Rinse ====== #
+
+        self.rinse_bv = Var(
+            initialize=5,
+            bounds=(2, 10),
+            doc="Number of bed volumes for rinse step",
+        )
+
+        self.rinse_flow = Var(
+            initialize=0.001,
+            units=pyunits.m**3 / pyunits.s,
+            doc="Rinse step rate",
+        )
+
+        self.t_rinse = Var(
+            initialize=360, units=pyunits.s, bounds=(120, 1200), doc="Rinse time"
+        )
+
+        self.main_pump_power = Var(
+            initialize=4e-6, units=pyunits.kW, doc="Main pump power [kW]"
+        )
+
+        self.regen_pump_power = Var(
+            initialize=4e-6, units=pyunits.kW, doc="Regen pump power [kW]"
+        )
+
+        self.bw_pump_power = Var(
+            initialize=4e-6, units=pyunits.kW, doc="Backwash pump power [kW]"
+        )
+
+        self.rinse_pump_power = Var(
+            initialize=4e-6, units=pyunits.kW, doc="Rinse pump power [kW]"
+        )
+
+        self.pump_efficiency = Var(
+            initialize=0.8,
+            units=pyunits.dimensionless,
+            bounds=(0, 1),
+            doc="Pump efficiency",
+        )
+
+        # Fix variables that should be fixed
+        self.resin_diam.fix()
+        self.resin_bulk_dens.fix()
+        self.bed_porosity.fix()
+        self.dimensionless_time.fix()
+        self.lh.fix()
+        self.regen_dose.fix()
+        self.regen_ww.fix()
+        self.regen_sg.fix()
+        self.t_bw.fix()
+        self.bw_rate.fix()
+        self.rinse_bv.fix()
+        self.pump_efficiency.fix()
+
```

Comment:
If these are to be fixed in this file, should they just be hard coded as numbers?  Otherwise they can be fixed in the test file (by the user).  Seems odd to fix vars in this file.

Reply:
I saw this in a different unit model but realized later the vars that were fixed were for regressions (and thus would likely not be changed by the user). I will eliminate these.